cmd/erasure-server-pool.go

```diff
@@ -335,7 +335,6 @@ func (z *erasureServerPools) getPoolIdxExistingWithOpts(ctx context.Context, buc
 			pinfo := poolObjInfo{
 				PoolIndex: i,
 			}
-			opts.VersionID = "" // no need to check for specific versionId
```

Comment:
changes in getPoolIdx* is breaking replication when server pools are in place

Reply:
@poornas this is fixed